Userland/Applications/Calendar/EventManager.cpp

```diff
@@ -26,13 +26,14 @@ OwnPtr<EventManager> EventManager::create()
 void EventManager::add_event(Event event)
 {
     m_events.append(move(event));
-    set_dirty(true);
+    m_dirty = true;
```

Comment:
What's the intention behind this commit? Either we should always use `set_dirty()`, or we should remove that method entirely.